modules/service/src/test/scala/ch/epfl/bluebrain/nexus/kg/service/io/JsonOpsSpec.scala

```diff
@@ -0,0 +1,60 @@
+package ch.epfl.bluebrain.nexus.kg.service.io
+
+import akka.http.scaladsl.model.Uri
+import ch.epfl.bluebrain.nexus.kg.service.io.RoutesEncoder.JsonOps
+import io.circe.Json
+import org.scalatest.{Inspectors, Matchers, WordSpecLike}
+
+class JsonOpsSpec extends WordSpecLike with Matchers with Inspectors {
+
+  private val contextUri    = Uri("http://localhost/v0/contexts/nexus/core/resource/v1.0.0")
+  private val contextString = Json.fromString("http://localhost/v0/contexts/nexus/core/resource/v1.0.0")
+
+  private val mapping = List(
+    Json.obj("@id"        -> Json.fromString("foo-id"), "nxv:rev" -> Json.fromLong(1)) ->
+      Json.obj("@context" -> contextString, "@id" -> Json.fromString("foo-id"), "nxv:rev" -> Json.fromLong(1)),
+    Json.obj("@context"   -> Json.fromString("http://foo.domain/some/context"),
+             "@id"        -> Json.fromString("foo-id"),
+             "nxv:rev"    -> Json.fromLong(1)) ->
+      Json.obj(
+        "@context" -> Json.arr(Json.fromString("http://foo.domain/some/context"), contextString),
+        "@id"      -> Json.fromString("foo-id"),
+        "nxv:rev"  -> Json.fromLong(1)
+      ),
+    Json.obj(
+      "@context" -> Json.arr(Json.fromString("http://foo.domain/some/context"),
+                             Json.fromString("http://bar.domain/another/context")),
+      "@id"     -> Json.fromString("foo-id"),
+      "nxv:rev" -> Json.fromLong(1)
+    ) ->
+      Json.obj(
+        "@context" -> Json.arr(Json.fromString("http://foo.domain/some/context"),
+                               Json.fromString("http://bar.domain/another/context"),
+                               contextString),
+        "@id"     -> Json.fromString("foo-id"),
+        "nxv:rev" -> Json.fromLong(1)
+      ),
+    Json.obj(
+      "@context" -> Json.obj("foo" -> Json.fromString("http://foo.domain/some/context"),
+                             "bar" -> Json.fromString("http://bar.domain/another/context")),
+      "@id"     -> Json.fromString("foo-id"),
+      "nxv:rev" -> Json.fromLong(1)
+    ) ->
+      Json.obj(
+        "@context" -> Json.obj("foo" -> Json.fromString("http://foo.domain/some/context"),
```

Comment:
A `@context` should be converted always to an array, except for when we didn't have previously any context. And what should be added is the link to "{{base}}/contexts/nexus/core/standards/v0.1.0".
So, input like
```json
"@context" {
"a": "http://localhost/a",
"b": "http://localhost/b"
}
```
to 
```json
"@context" [
{
  "a": "http://localhost/a",
  "b": "http://localhost/b"
},
"{{base}}/contexts/nexus/core/standards/v0.1.0"
]
```